bins/broker/session/Session.h

```diff
@@ -111,6 +111,7 @@ class Session {
   const Connection &_connection;
   mutable std::atomic_int _txCounter;
   mutable CircularQueue<State> _stateStack;
+  mutable std::recursive_mutex _rebegin_mutex;
```

Comment:
please, rename it to _rebeginMutex

Reply:
Try to use .clang-tidy from repository for naming conventions.

Reply:
Oh yes, of course!